dashboard/src/components/ChartList/ChartList.tsx

```diff
@@ -17,6 +20,20 @@ class ChartList extends React.Component<IChartListProps> {
   }
 
   public render() {
+    if (!this.props.charts.items) {
```

Comment:
we should check for isFetching also because otherwise this message will flash before charts have loaded right?